### ISSUE TITLE
Python3-related issues in ngamsUtils/ngamsUtils/ngasCheckFileCopies.py

### ISSUE THREAD

**Issue by ulange-eso-org** [ICRAR/ngas#95]
```
--- a/src/ngamsUtils/ngamsUtils/ngasCheckFileCopies.py
+++ b/src/ngamsUtils/ngamsUtils/ngasCheckFileCopies.py
@@ -111,7 +111,7 @@ def check_copies(disk_id, notification_email):
         glob_file_dict[file_key].append(file_info)

     # Order the list according to (1) Number of copies and (2) Alphabetically
-    file_key_list = glob_file_dict.keys()
+    file_key_list = list(glob_file_dict.keys())
     file_key_list.sort()
     sort_file_dict = {}
     for file_key in file_key_list:
@@ -127,11 +127,11 @@ def check_copies(disk_id, notification_email):
     message_format = "{:60.60s} {:7.7s} {:5.5s} {:4.4s}\n"
     report += message_format.format("File ID", "Version", "Total", "Good")
     report += 50 * "-" + "\n"
-    no_file_key_list = sort_file_dict.keys()
+    no_file_key_list = list(sort_file_dict.keys())
     no_file_key_list.sort()
     for no_file_key in no_file_key_list:
         no_file_key_dict = sort_file_dict[no_file_key]
-        file_key_list = no_file_key_dict.keys()
+        file_key_list = list(no_file_key_dict.keys())
         file_key_list.sort()
         for file_key in file_key_list:
             total_copies = 0
@@ -151,7 +151,7 @@ def check_copies(disk_id, notification_email):
     print("\n" + report)

     if notification_email:
-        ngasUtilsLib.send_email("ngasCheckFileCopies: FILE COPIES CHECK REPORT (%s)".format(disk_id),
+        ngasUtilsLib.send_email("ngasCheckFileCopies: FILE COPIES CHECK REPORT ({0})".format(disk_id),
                                 notification_email, report)


@@ -187,7 +187,7 @@ def main():
                 sys.exit(1)
             index += 1
         except Exception as e:
-            print("\nProblem executing the tool: %s\n".format(str(e)))
+            print("\nProblem executing the tool: {0}\n".format(str(e)))
             print(correct_usage())
             sys.exit(1)
     if notification_email is None:
```

**Comment by rtobar** [comment]
Thanks @ulange-eso-org for the new set of reports and suggested fixes, and sorry for not attending them until now, as I was on leave this week. I've addressed all the issues (I think) and have pushed all commits to the `uwe-fixes` branch. Let me know if you're happy with them, and whether you'd want to merge the branch back to `master` (or if you'd rather wait a bit more).

In the meanwhile I also fixed the CI builds (the `requests` package, which we use, issued a new release where they removed support for 2.7/3.6) and merged that to the `uwe-fixes` branch too, so the CI runs should be green now.
